Source/WebKit/Shared/haiku/ProcessExecutablePathHaiku.cpp

```diff
@@ -29,25 +29,28 @@
 
 #include <Entry.h>
 #include <String.h>
+#include <wtf/NeverDestroyed.h>
 
 namespace WebKit {
+using namespace std;
 
 String executablePathOfWebProcess()
 {
-    return BString("./bin/WebProcess");
+	static NeverDestroyed<const String> WebKitWebProcessName("./bin/WebProcess");
```

Comment:
still not using find_directory, environment variables, etc as it should. At least add a TODO comment to remember.

Reply:
I used entry ref I guess I can't find it now where is it. for now since we are testing I stopped with minibrowser later we will add all the things.
Sure I will add a todo